content/docs/06-integrations/00-antrea-cni.md

```diff
@@ -0,0 +1,106 @@
+metaTitle: 'Antrea CNI'
+metaDescription: 'Antrea CNI network pack for Palette Kubernetes Clusters'
+hiddenFromNav: true
+type: "integration"
+category: ['network']
+logoUrl: 'https://registry.spectrocloud.com/v1/palette-upgrader/blobs/sha256:b6081bca439eeb01a8d43b3cb6895df4c088f80af978856ddc0da568e5c09365?type=image/png'
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Antrea CNI
+
+Palette supports Antrea controller network interface (CNI) for Kubernetes clusters. Antrea  CNI enables each pod to have exclusive IP addresses from the subnet with direct accessibility. 
+
+Antrea leverages Open vSwitch to implement pod networking and security features. Open vSwitch enables Antrea to implement Kubernetes network policies efficiently.
+
+# Supported Versions
+
+**1.9.x**
+
+## Prerequisites
+
+- Enable the ``enableNodeIPAM`` integrated NodeIPAM controller within the Antrea controller.
+- When deploying a cluster using kubeadm, specify the ``--pod-network-cidr <cidr>`` option and provide the IP address with the classless inter-domain routing (CIDR). For example: 
+
+    ``--pod-network-cidr=10.244.0.0/16``
+
+    <br />
+
+    <WarningBox>
+
+    To avoid overlapping your pod network with any of your host networks, you should think of a suitable CIDR block to specify if you deploy a cluster using ``kubeadm init`` with ``--pod-network-cidr <cidr>`` or as a replacement in your network plugin's YAML.
+
+    </WarningBox>
+
+- The Open vSwitch kernel module must be present on every Kubernetes node.
+
+## Parameters
+
+The Antrea CNI pack supports the following parameters.
+
+| Parameter | Description | Required (Y/N) |
+|-----------|-------------|---------|
+| nodeIPAM:enable | This enables the integrated NodeIPAM controller within the Antrea controller. The default is `false`. | Y |
+| clusterCIDRs | CIDR ranges for pods in the cluster. The CIDRs could be either IPv4 or IPv6. You can specify up to one CIDR for each IP family. | N |
+| serviceCIDRv6 | IPv6 CIDR ranges reserved for Services. | N |
+| nodeCIDRMaskSizeIPv4 | Mask size for IPv4 Node CIDR in IPv4 or dual-stack cluster. | N |
+| nodeCIDRMaskSizeIPv6 | Mask size for IPv6 Node CIDR in IPv6 or dual-stack cluster. | N |
+| NodeIPAM | The feature toggle for ``antrea-controller``. The default is `false`. If you use CIDR ranges, set this to ``true``.  | N |
+| ServiceExternalIP | The feature toggle for ``antrea-agent`` and ``antrea-controller``. If you use the LoadBalancer service, set this to ``true``. | N |
+
+
+## Usage
+
+Kubernetes network policies are supported by default.
+
+Antrea supports LoadBalancer services. Typically, implementing LoadBalancer services requires an external load balancer that is implemented by the Kubernetes Cloud Provider. 
```

Comment:
```suggestion
Antrea supports LoadBalancer services. Typically, implementing LoadBalancer services requires an external load balancer that is implemented by the Kubernetes cloud provider. 
```
the cloud provider feels odd as a proper noun here when it's used to reference all cloud providers.